cmd/kubeadm/app/preflight/checks.go

```diff
@@ -560,6 +561,19 @@ func (sysver SystemVerificationCheck) Check() (warnings, errorList []error) {
 	return warns, nil
 }
 
+func MessageLogger(msgs []system.LogMessage) {
```

Comment:
this func is a way we could use the new `LogMessage` struct in kubeadm. but I figure we need a better spot to put then it and was unsure.